data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepperExecute.java

```diff
@@ -30,7 +34,17 @@ public static void main(String[] args) {
         }
         final DataPrepper dataPrepper = DataPrepper.getInstance();
         if (args.length > 0) {
-            dataPrepper.execute(args[0]);
+            String configurationFileLocation = args[0];
+            if (args[0].endsWith(".conf")) {
+                LogstashConfigConverter logstashConfigConverter = new LogstashConfigConverter();
+                try {
+                    configurationFileLocation = logstashConfigConverter.convertLogstashConfigurationToPipeline(
+                            args[0], String.valueOf(Paths.get("data-prepper-core/build/libs/")));
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
We should be using SLF4J logging for this. Somewhat like: `LOG.error("custom error message", ex)`.